study.md

```diff
@@ -134,41 +134,42 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
-```
+Http is a protocol using standarized way to communication to obtain information.```
 
 ## Describe what a client is and what a server is
 
  What is a client and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
-```
+Clients makes request to the server, which stores the data and answers requests.```
 
 ## Describe the 4 most common HTTP verbs
 
 What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
-```
+GET: instructs the server to transmit the data,
+
+PUT: updating or creating the resource in the body
+DELETE:delete the resource
+ POST:used to trigger operations on the server
+ ```
 
 ## Describe what a Response is
 
 What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
-```
+response is an action to a request.```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+curl -v GET http://reddit.com
```

Comment:
Don't think this is right, it should look more like this:

`curl --request GET https://www.reddit.com`